config/email.js

```diff
@@ -0,0 +1,30 @@
+const nodemailer = require('nodemailer');
+const path = require('path');
+require('dotenv').config();
```

Comment:
`require('dotenv').config():` you could add this line to the main server file (app.js, server.js) so you won't need to include it on each file where you need env variables